src/app/pages/AccountPage/index.tsx

```diff
@@ -48,6 +48,13 @@ const StyledNavItem = styled(NavLink)`
   &.active {
     background-color: ${({ theme }) => normalizeColor('background-back', theme)};
   }
+
+  @media only screen and (min-width: ${({ theme }) => `${theme.global?.breakpoints?.small?.value}px`}) {
+  }
```

Comment:
Is this leftover, or a placeholder for future?

Reply:
leftover